src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -17,7 +17,7 @@ function isUserTyping(str) {
 window.isUserTyping = isUserTyping;
 
 const loadUpFunction = function () {
-    const typingErrMsg = 'Please use | character ( shift+\\ ) to seperate fields, or copy from your existing spreadsheet.';
+    const typingErrMsg = 'Please use | character ( shift+\\ ) to separate fields, or copy from your existing spreadsheet.';
```

Comment:
Is this and the `isUserTyping` function above still relevant for the new spreadsheet interface?

Reply:
Not relevant anymore, removed all unrelevant old code :smile: